### ISSUE TITLE
D10 Testing: Accessibility

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14545]
## Description
In prep for the upgrade to Drupal 10, an accessibility review of the CMS should be done to make sure that no launch-blocking accessibility issues will be introduced with the upgrade. Any new accessibility issues found, launch-blocking or otherwise should be ticketed to be reviewed and prioritized as needed.

## Acceptance Criteria
- [x] Accessibility review of D10 environment is completed
- [x] Tickets are created for any new issues found
- [x] PO/PM are notified of any launch-blocking issues (if any)

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`
